cmd/zed/ztests/no-lake-root.yaml

```diff
@@ -1,7 +1,7 @@
 script: |
-  ! zed lake ls
+  ! zed ls
```

Comment:
```suggestion
  ! zed ls -lake ''
```
Because the default lake should be http://localhost:9867.

Reply:
Didn't know this was the case. Ok.